engine-comparison/dispatcher.sh

```diff
@@ -81,15 +92,17 @@ build_benchmark_using() {
 handle_benchmark() {
   BENCHMARK=$1
   FENGINE_CONFIG=$2
-  THIS_BENCHMARK=${BENCHMARK}-with-$(basename ${FENGINE_CONFIG}) # Just for convenience
+  # Just for convenience
+  FENGINE_NAME=$(basename ${FENGINE_CONFIG})
+  THIS_BENCHMARK=${BENCHMARK}-with-${FENGINE_NAME}
 
   build_benchmark_using $BENCHMARK $FENGINE_CONFIG $THIS_BENCHMARK
-  gsutil rsync -r $SEND_DIR ${GSUTIL_BUCKET}/binary-folders/$THIS_BENCHMARK
+  gsutil -m rsync -rd $SEND_DIR ${GSUTIL_BUCKET}/binary-folders/$THIS_BENCHMARK
   # GCloud instance names must match the following regular expression:
   # '[a-z](?:[-a-z0-9]{0,61}[a-z0-9])?'
   INSTANCE_NAME=$(echo "fts-runner-${THIS_BENCHMARK}" | \
     tr '[:upper:]' '[:lower:]' | tr -d '.')
-  create_or_start $INSTANCE_NAME ./FTS/engine-comparison/runner-startup-script.sh
+  create_or_start $INSTANCE_NAME "benchmark=${BENCHMARK},fengine=${FENGINE_NAME}" "startup-script=$WORK/FTS/engine-comparison/startup-runner.sh"
```

Comment:
To me, this is a tradeoff with having `create_or_start` be very reusable. Really, though, it is only called twice; there's no chance of using these specific metadata tags for `begin-experiment`, but there's also little chance of needing any metadata for `begin-experiment`. Maybe there's no need for `create_or_start` to be so "generic"